OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -40,6 +40,8 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 		public RallyPointInfo Info;
 		public string PaletteName { get; private set; }
 
+		private const uint ExtraDataSetPrimaryBuilding = 1;
```

Comment:
I would rather just have it as a 1 since it is in all in one class but I defer this to @pchote or another one of the main developers.